src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigCol {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, Color color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double getFirstLeg(double firstLeg) {
+        return firstLeg;
+    }
+
+    public double getSecondLeg(double secondLegLeg) {
```

Comment:
The getter method 'getSecondLeg' has a typo in its parameter name and should not take any parameters.